### PR TITLE
Add jupyterhub magpie auth

### PR DESCRIPTION
**Pull request by davidcaron**
And change jupyterhub image to 1.0.0 (related to jupyterhub/jupyterhub#2852)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
-FROM jupyterhub/jupyterhub:1.0
+FROM jupyterhub/jupyterhub:1.0.0
```
Comment (tlvu) [inline Dockerfile]: Thanks for pinning that.  1.1.0 is available if you feeling lucky :D.  I am fine with 1.0.0.

**Comment by tlvu** [comment]
@davidcaron why tag `1.1-date`, the base image is still `1.0.0` !

**Comment by davidcaron** [comment]
Ok, I wasn't sure if the tag would follow our own convention or the jupyterhub docker image. I'll revert that.

**Comment by tlvu** [comment]
We want to keep the jupyterhub version, see https://github.com/Ouranosinc/jupyterhub/blob/master/README.md#release-instruction "Keep the same original upstream `jupyterhub/jupyterhub` version in the tag so we know what version our image is based on."

**Comment by tlvu** [comment]
The original upstream tag is `1.0.0`, not `1.0`, so we should tag `1.0.0-date` and not `1.0-date`.

**Comment by davidcaron** [comment]
👌

**Pull request merged**